src/components/CodeView/index.tsx

```diff
@@ -140,6 +145,13 @@ export class CodeViewBase extends React.Component<Props> {
 
     const shouldHighlight =
       !codeWasTrimmed && _codeCanBeHighlighted({ code: codeLines });
+    const highlightAlert = !shouldHighlight ? (
+      <Alert className={styles.highlightAlert} variant="warning">
+        {gettext(
+          'Syntax highlighting was disabled, and some lines may have been truncated, for performance.',
```

Comment:
This makes me think that we should maybe mark the lines that are truncated. Can we append a string to each line maybe? Something like ` /* truncated */` maybe?

Reply:
That's a good idea, thanks!